package.json

```diff
@@ -46,7 +48,7 @@
     ]
   },
   "devDependencies": {
-    "@edma/design-tokens": "^4.2.5",
+    "@edma/design-tokens": "^4.3.0",
```

Comment:
Double check the Design Token docs to make sure we didn't accidentally break anything by updating this.

Reply:
4.3.0 seems to be the correct version based on the tokens repo